### PR TITLE
Generate golden log/object fullalloydbcluster test

### PR DESCRIPTION
**Pull request by anhdle-sso**
### Change description

Generate golden log for fullalloydbcluster and normalize logic for alloydb

### Tests you have done

- [x] Run `make ready-pr` to ensure this PR is ready for review.
- [x] Perform necessary E2E testing for changed resources.

### CODE REVIEW

**Comment by anhdle-sso** [comment]
/cc @yuwenma 

Other 2 tests golden logs have too many fields to normalize so I do not submit it.

**Comment by yuwenma** [comment]
/lgtm 

Looks like the replacement in unified_test.go has bigger impact. 😄 I guess you don't need the `user` `address` `targetId`.

**Comment by anhdle-sso** [comment]
I double check, we still need _user_ and _address_

**Comment by yuwenma** [comment]
It looks like the presubmit is running against a different github actions than what this branch supposes to be (which makes me doubt if some other commits could be missing). i.e. You can check this file https://github.com/anhdle-sso/k8s-config-connector/blob/regen_cluster_golden/scripts/github-actions/tests-e2e-direct.sh which does not include the `cloudbuildworkerpool` the presubmit runs.

Could you fetch and rebase to upstream master?

File: pkg/test/resourcefixture/testdata/basic/alloydb/v1beta1/alloydbcluster/fullalloydbcluster/_generated_object_fullalloydbcluster.golden.yaml [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+    - TUESDAY
+    - WEDNESDAY
+    - THURSDAY
+    - FRIDAY
+    - SATURDAY
+    - SUNDAY
+  databaseVersion: POSTGRES_15
+  encryptionInfo:
+  - encryptionType: CUSTOMER_MANAGED_ENCRYPTION
+  name: projects/${projectId}/locations/us-central1/clusters/alloydbcluster-${uniqueId}
```
Comment (yuwenma) [inline pkg/test/resourcefixture/testdata/basic/alloydb/v1beta1/alloydbcluster/fullalloydbcluster/_generated_object_fullalloydbcluster.golden.yaml:91]: Is this from the TF based resource?

**Comment by yuwenma** [comment]
/lgtm
/approve

Thanks!

**Pull request merged**